nixos/modules/services/x11/window-managers/qtile.nix

```diff
@@ -1,23 +1,63 @@
-{ config, lib, pkgs, ... }:
+{ config, pkgs, lib, ... }:
 
 with lib;
 
 let
   cfg = config.services.xserver.windowManager.qtile;
+  pyEnv = pkgs.python3.withPackages (p: [ cfg.package ] ++ (cfg.extraPackages p));
```

Comment:
Do we need to worry about double-wrapping here?

If so, perhaps the default `cfg.package` should be `qtile-unwrapped` or this could optionally do `(cfg.package.unwrapped or cfg.package)` to get the unwrapped derivation, to allow either wrapped or unwrapped to be passed in

Reply:
Thanks! I decided to do both the `cfg.package` and `(cfg.package.unwrapped or cfg.package)` change, as they both made sense to me.